sdk/tables/Azure.Data.Tables/tests/perf/Program.cs

```diff
@@ -1,15 +1,29 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Reflection;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
 using BenchmarkDotNet.Running;
 
 namespace Azure.Data.Tables.Performance
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
-             BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
+            // Allow the framework to execute the test scenarios.
+            await PerfProgram.Main(Assembly.GetEntryAssembly(), args);
+
+            /*
```

Comment:
nit:  Did you mean to include this in the PR?   Looks a bit like a personal note.